### PR TITLE
Add image classification notebook

### PR DESCRIPTION
**Pull request by NielsRogge**
Quick preview: https://colab.research.google.com/drive/12LV6Vd14ug2KfbCv3ariiOQpJlJfpjck?usp=sharing

### CODE REVIEW

File: examples/image_classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,5278 @@
+{
```
Comment (sgugger) [inline examples/image_classification.ipynb:1]: <p>Input your token (login with username and password is deprecated now).</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Comment (sgugger) [inline examples/image_classification.ipynb:1]: <p>Formatting here could be better for the imports.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

**Review by sgugger — APPROVED** [review]
Thanks for adding this! Left a couple of comments in ReviewNB, let me know if you can't see them.

File: examples/image_classification.ipynb (hunk 1 above)
Comment (nateraw) [reply examples/image_classification.ipynb:1]: <p>+1</p>

Comment (nateraw) [inline examples/image_classification.ipynb:1]: <p>Would suggest updating to:</p><p> </p><pre class="ql-syntax" spellcheck="false">%%capture
! pip install git+https://github.com/huggingface/huggingface_hub.git@main
! sudo apt -qq install git-lfs
! git config --global credential.helper store
</pre>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Comment (nateraw) [inline examples/image_classification.ipynb:1]: <p>remove this if you do changes I showed above in installation step</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Comment (nateraw) [inline examples/image_classification.ipynb:1]: <p>(1) <em>resizing ...</em>(2) <em>normalizing</em> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='32'/>

Comment (nateraw) [inline examples/image_classification.ipynb:1]: <p>I think you can get rid of the last sentence....it's not really saying it "in other words" 😅</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='36'/>

Comment (nateraw) [inline examples/image_classification.ipynb:1]: <p>All other arguments are <strong>not </strong> optional. You should add a note about the <code>removed_unused_columns=False</code>  situation going on here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>

Comment (nateraw) [inline examples/image_classification.ipynb:1]: <p>Nice thx for mentioning </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>

Comment (nateraw) [inline examples/image_classification.ipynb:1]: <p>Nitpick - you could do more when calling train and evaluate. consider updating to use the following two codeblocks instead (but do whatever you want)</p><p> </p><pre class="ql-syntax" spellcheck="false">train_results = trainer.train()
trainer.save_model()
trainer.log_metrics("train", train_results.metrics)
trainer.save_metrics("train", train_results.metrics)
trainer.save_state()
</pre><p> </p><pre class="ql-syntax" spellcheck="false">metrics = trainer.evaluate(prepared_ds['validation'])
trainer.log_metrics("eval", metrics)
trainer.save_metrics("eval", metrics)
</pre><p> </p><p>Then push to hub</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='55'/>

Comment (nateraw) [inline examples/image_classification.ipynb:1]: <p>lmao love it</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/notebooks/pull/143/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

**Review by nateraw — APPROVED** [review]
LGTM I just left a few nitpicks with ReviewNB. thanks 🚀 

Lets not forget to update this whenever we make any major changes to the image classification script (since this is basically that in nb form).

**Pull request merged**